src/hotspot/share/cds/archiveBuilder.hpp

```diff
@@ -212,6 +212,7 @@ class ArchiveBuilder : public StackObj {
   ResizeableResourceHashtable<address, address, AnyObj::C_HEAP, mtClassShared> _buffered_to_src_table;
   GrowableArray<Klass*>* _klasses;
   GrowableArray<Symbol*>* _symbols;
+  unsigned int _entropy_seed;
```

Comment:
Shouldn't it be a static const?

Reply:
I think it is meant to be a member of the (one and only existing) archive builder.